src/main/java/emissary/place/ServiceProviderRefreshablePlace.java

```diff
@@ -78,21 +90,49 @@ public ServiceProviderRefreshablePlace(final ServiceProviderRefreshablePlace pla
         setupPlace(null, place.placeLocation, register);
     }
 
+    @Override
+    protected void setupPlace(@Nullable final String theDir, final String placeLocation, final boolean register) throws IOException {
+        super.setupPlace(theDir, placeLocation, register);
+
+        try {
+            Preconditions.checkNotNull(this.configG, "The configurator is null");
+            final var path = configG.findStringEntry("MONITORING_PATH");
+            if (StringUtils.isNotBlank(path)) {
+                final var intervalMinutes = configG.findLongEntry("MONITORING_INTERVAL_MINUTES", 15);
+                this.monitor = new Monitor(this, path, intervalMinutes);
```

Comment:
I think to get started, having an info log message here would be nice so we know it wired correctly. Once we convert things over we can change to debug.

Reply:
I'm good with a follow-on for this addition.